charts/app-config/values-production.yaml

```diff
@@ -87,11 +87,6 @@ govukApplications:
             key: MONGODB_URI
       - name: REDIS_URL
         value: redis://shared-redis-govuk.eks.production.govuk-internal.digital
-      - name: SECRET_KEY_BASE
```

Comment:
Oh that's awesome that this gets rids of that bit of copypasta in the app values :D